src/core/electric_utility.jl

```diff
@@ -101,3 +196,121 @@ function ElectricUtility(;
         )
     end
 end
+
+function region(region_abbr::String)
+    lookup = Dict(
+        "AK" => "Alaska",
+        "CA" => "California",
+        "EMW" => "Great Lakes / Atlantic",
+        "NE" => "Northeast",
+        "NW" => "Northwest",
+        "RM" => "Rocky Mountains",
+        "SC" => "Lower Midwest",
+        "SE" => "Southeast",
+        "SW" => "Southwest",
+        "TX" => "Texas",
+        "WMW" => "Upper Midwest",
+        "HI" => "Hawaii (except Oahu)",
+        "HI-Oahu" => "Hawaii (Oahu)"
+    )
+    try
+        return lookup[region_abbr]
+    catch
+        return "None"
```

Comment:
maybe "not found" or `missing` instead of "None"? how will downstream code handle this "None"?

Reply:
I changed the default emission region to `nothing` because that is what is expected in the `emissions_series` function. However, the downstream code, specifically the code in `add_yr1_emissions_calcs` will fail if any emission series is `nothing`. We need to fix this by either throwing an error, not modeling emissions with a warning, or setting the particular emission series to zeros with a warning.

Reply:
if emissions factor look up in emissions_series fails, we are now warning and returning zeros